infrastructure/charts/agent/templates/apisixroute.yaml

```diff
@@ -64,6 +69,11 @@ spec:
       enable: true
       config:
         regex_uri: ["^/prism-agent/didcomm(.*)", "/$1"]
+    - name: request-id
+      enable: true
+      config:
+        header_name: "X-Request-ID"
+        include_in_response: true
     {{ template "cors" . }}
     {{ template "headers.requestId" . }}
```

Comment:
@FabioPinheiro The request ID is actually already enabled under this template - so all the new additions should be uneeded

Reply:
https://github.com/hyperledger-labs/open-enterprise-agent/blob/main/infrastructure/charts/agent/templates/_helpers.tpl

Reply:
Thanks
I removed them